pkg/apis/networking/validation/validation.go

```diff
@@ -602,3 +602,85 @@ func allowInvalidWildcardHostRule(oldIngress *networking.Ingress) bool {
 	}
 	return false
 }
+
+// ValidateClusterCIDRConfigName validates that the given name can be used as an
+// ClusterCIDRConfig name.
+var ValidateClusterCIDRConfigName = apimachineryvalidation.NameIsDNSLabel
```

Comment:
Make sure you only export names that really need to be called outside of the package.

Reply:
Yes this needs to be accessible from `k8s.io/kubernetes/pkg/apis/core/validation` package